tests/v3_api/test_import_cluster.py

```diff
@@ -0,0 +1,131 @@
+
+    # Create RKE K8s Cluster
+    execute_rke_cmd("up", clusterfilepath)
+
+    # Get the cluster Id
+    clustername = random_test_name("testimport")
+    cluster_id_url = CATTLE_TEST_URL + "/v3/cluster?_replace = true"
+    data_test1 = {"dockerRootDir": "/var/lib/docker",
+                  "enableNetworkPolicy": False, "type": "cluster",
+                  "name": clustername}
+    headers = {"Content-Type": "application/json",
+               "Accept": "application/json",
+               "Authorization": "Bearer " + ADMIN_TOKEN}
+    response = requests.post(cluster_id_url, json=data_test1,
+                             verify=False, headers=headers)
+
+    print(response.status_code)
+    assert response.status_code == 201
+    assert response.content is not None
+    print(response.content)
+    json_response = json.loads(response.content)
+    clusterid = json_response["id"]
+    print(clusterid)
+
+    # Obtain the URL for importing a cluster
+    cluster_import_url = CATTLE_TEST_URL + "/v3/clusterregistrationtoken"
+    print("CLUSTER IMPORT URL")
+    print(cluster_import_url)
+
+    data_test2 = {"type": "clusterRegistrationToken", "clusterId": clusterid}
+
+    response = requests.post(cluster_import_url, json=data_test2,
+                             verify=False, headers=headers)
+
+    assert response.status_code == 201
+    assert response.content is not None
+    json_response = json.loads(response.content)
+    command = json_response["insecureCommand"]
+    print("Insecure Command")
+    print(command)
+    rke_config_file = "kube_config_cluster.yml"
+    finalimportcommand = command + " --kubeconfig " + DATA_SUBDIR + "/" +\
+        rke_config_file
+    print("Final command to import cluster is:")
+    print(finalimportcommand)
+    result = run_command(finalimportcommand)
+    print(result)
+
+    clusters = client.list_cluster(name=clustername).data
+    assert len(clusters) > 0
+    print("Cluster is")
+    print(clusters[0])
+
+    # Validate the cluster
+    cluster = validate_cluster(client, clusters[0],
+                               check_intermediate_state=False)
+
+    if RANCHER_CLEANUP_CLUSTER == "True":
+        delete_cluster(client, cluster)
+
+
+def create_nodes():
+
+    aws_nodes = \
+        AmazonWebServices().create_multiple_nodes(
+            3, random_test_name("testcustom"), wait_for_ready=True)
+    assert len(aws_nodes) == 3
+    for aws_node in aws_nodes:
+        print(aws_node)
+        print(aws_node.public_ip_address)
+
+    return aws_nodes
+
+
+def create_rke_cluster_config(aws_nodes):
+
+    configfile = "cluster.yml"
+
+    rkeconfig = readDataFile(DATA_SUBDIR, configfile)
+    rkeconfig = rkeconfig.replace("$ip1", aws_nodes[0].public_ip_address)
+    rkeconfig = rkeconfig.replace("$ip2", aws_nodes[1].public_ip_address)
+    rkeconfig = rkeconfig.replace("$ip3", aws_nodes[2].public_ip_address)
+
+    print(rkeconfig)
+    clusterfilepath = DATA_SUBDIR + "/" + "cluster.yml"
```

Comment:
@soumyalj Instead of replacing the existing template mL file , can you create a new one and use it ?